.travis.yml

```diff
@@ -57,10 +57,7 @@ install:
   - yarn install --force
 
 script:
-  - yarn lint
-  - yarn build:prod
-  - yarn test:setup
-  - yarn test
+  - yarn lint && yarn build:prod && yarn test:setup && yarn test
```

Comment:
Unlike the other CI platforms, Travis happily marched on after it encountered a linting issue: https://travis-ci.org/desktop/desktop/builds/326538605#L717

This is the documented way to make the build fail fast: https://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Build-Step